postgres-appliance/configure_spilo.py

```diff
@@ -257,7 +257,7 @@ def get_placeholders(provider):
     placeholders.setdefault('WALE_BACKUP_THRESHOLD_PERCENTAGE', 30)
     placeholders.setdefault('WALE_ENV_DIR', os.path.join(placeholders['PGHOME'], 'etc', 'wal-e.d', 'env'))
 
-    if provider in (PROVIDER_AWS, PROVIDER_GOOGLE):
+    if provider in (PROVIDER_AWS, PROVIDER_GOOGLE, PROVIDER_LOCAL):
```

Comment:
We don't really want placeholder.setdefault('USE_WALE', True) for the PROVIDER_LOCAL, since WAL-E is not able when you run in minukube. I suggest instead to move USE_K8S after the block in lines 260: 275, which should set a callback script for K8S not depending on a current provider.